_events/20221024_acat-poster-bellecaching.yml

```diff
@@ -0,0 +1,18 @@
+---
+title: XRootD caching for Belle II
+date: 2022-10-24 12:00 +0200
+date_end: 2022-10-28 12:00 +0200
+presenter: Moritz Bauer
+author:
+  - Moritz Bauer
+    Gunter Quast
+    Manuel Giffels
+    Matthias Schnepf
+    Max Fischer
```

Comment:
This looks wrong but apparently [we did it like that before](https://github.com/MatterMiners/matterminers.github.io/blob/master/_events/20191107_chep-talk2.yml). Shouldn't this be a proper YAML list, i.e. *every* item with a leading `-`? Loading this with PyYAML gives a list of a single string:

```
'co-author': ['Max Fischer Andreas Heiss Eileen Kuehn Matthias Schnepf Ralf Florian von Cube Andreas Petzold Guenter Quast'],
```

Did we just do that wrong the first time around as well, @eileen-kuehn @giffels?

Reply:
Looks strange indeed. I would prefer as well:

```yaml
author:
  - Foo Bar
  - Chuck Norris
  - ...
```

Reply:
I was unsure too, but did it that way because that's how it was at previous events. I'll change it for those six files. It might not be an issue, though, as we do not use this variable anywhere in our templates?